beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -1893,6 +1895,111 @@ func (s *Server) GetProposerLookahead(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// GetBlobs retrieves blobs for a given block id.
+func (s *Server) GetBlobs(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "beacon.GetBlobs")
+	defer span.End()
+
+	segments := strings.Split(r.URL.Path, "/")
+	blockId := segments[len(segments)-1]
+
+	var verifiedBlobs []*blocks.VerifiedROBlob
+	var rpcErr *core.RpcError
+
+	// Check if versioned_hashes parameter is provided
+	versionedHashesStr := r.URL.Query()["versioned_hashes"]
+	versionedHashes := make([][]byte, 0, len(versionedHashesStr))
```

Comment:
```suggestion
	versionedHashes := make([][]byte, len(versionedHashesStr))
```
The length will always be equal to `len(versionedHashesStr)`

Reply:
i need to change append to setting then

Reply:
i updated